### PR TITLE
move: example for moving between dirs

### PR DESCRIPTION
**Pull request by pared**
Related to https://github.com/iterative/dvc/pull/1505

### CODE REVIEW

File: static/docs/commands-reference/move.md [hunk 1]
```diff
@@ -106,4 +106,34 @@ To track the changes with git run:
 
 	git add .gitignore .gitignore data.csv.dvc  
 ```
+Here we use `dvc add` to put a file under DVC control. Then we use `dvc move` to change data directory. Note, that data stage file is also moved. If target path already exists and is a directory, data file is moved with unchanged name into this folder.
+
+```
```
Comment (shcheklein) [inline static/docs/commands-reference/move.md]: put `dvc` right after triple accent sign to enable syntax highlight

File: static/docs/commands-reference/move.md [hunk 2]
```diff
@@ -106,4 +106,34 @@ To track the changes with git run:
...
+
+    $ dvc move data/foo data2/subdir/
+    $ tree
+.
+├── data
+└── data2
+    └── subdir
+        ├── foo
+        └── foo.dvc
+
```
Comment (shcheklein) [inline static/docs/commands-reference/move.md]: Love the visualization!!

File: static/docs/commands-reference/move.md (hunk 1 above)
Comment (pared) [reply static/docs/commands-reference/move.md]: dvc or text?

Comment (shcheklein) [reply static/docs/commands-reference/move.md]: `dvc` - we have a custom highlighter. Check the `run.md` fo the reference.

**Comment by pared** [comment]
AFAIK move can be applied both to file and directory, do you think I should make some description how does it behave in both cases?

**Comment by efiop** [comment]
@pared That would be great!

**Comment by pared** [comment]
How about now? @shcheklein? @efiop

**Comment by shcheklein** [comment]
> How about now? @shcheklein? @efiop?

Looks great. Please, format the text to be under 80 symbols. It's a convention for the documentation.

**Comment by shcheklein** [comment]
Good stuff. Let's merge it!

**Pull request merged**